integrations/docker/images/chip-build-bouffalolab/Dockerfile

```diff
@@ -1,9 +1,11 @@
 ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 
-RUN set -x \
-    && pip install bflb-iot-tool \
-    && apt-get update \
-    && apt-get install -fy \
-    vim \
-    && : # last line
+RUN apt update -y \
```

Comment:
since this changes dockerfile content (both bouffalolab and vscode) please update the version in chip-build/version with a new version and explanation on why the change was made.

Reply:
Version and description updated